src/com/sun/ts/tests/common/vehicle/ejb3share/UserTransactionWrapper.java

```diff
@@ -24,10 +24,10 @@
 
 import javax.persistence.EntityTransaction;
 import javax.persistence.PersistenceException;
-import jakarta.transaction.NotSupportedException;
-import jakarta.transaction.SystemException;
-import jakarta.transaction.UserTransaction;
-import jakarta.transaction.Status;
+import javax.transaction.NotSupportedException;
```

Comment:
@Tomas-Kraus it looks like you included your commit that reverts your javax.transaction changes, I don't think you want to include that, as it is reverting the changes I merged yesterday from my pr that I started previously and was already reviewed.

Reply:
OK, will remove this from PR

Reply:
`kratz@mbp:/data/jakartaee-tk>git diff ee4j/master | grep '+import javax.transaction'`
returns nothing now. :)